.github/workflows/lint.yml

```diff
@@ -81,17 +81,19 @@ jobs:
         run: |
           set -eux
           mypy | sort | grep -v '^Found ' > typeerrors-HEAD.txt
-          git checkout -f master
+          forkpoint=$(git merge-base master HEAD)
+          git checkout -f $forkpoint
           mypy | sort | grep -v '^Found ' > typeerrors-MASTER.txt
           diff -u typeerrors-MASTER.txt typeerrors-HEAD.txt | grep -v '^+++' | grep '^+' > typeerrors-CHANGED.txt || true
           cat typeerrors-CHANGED.txt | sed 's/^+//' | sed 's/: error:/:1: mypy:/' | tee ${GITHUB_WORKSPACE}/output-annotations.txt
       - name: Add annotations
-        uses: pytorch/add-annotations-github-action@master
+        uses: stephenroller/add-annotations-github-action@master
```

Comment:
Just checked out your forked version - adding in a warning level is really useful! Will you see if you can get it merged into PyTorch?

Reply:
I'd like that, waiting on a review from the upstream maintainer.